firmware/arduino/chessboard/circleFunction.ino

```diff
@@ -259,3 +272,72 @@ float getInstantaneousSlope(int radius, int xStepsRemaining, int yStepsRemaining
     
     return ((float) radius - xStepsRemaining) / yStepsRemaining;
 }
+
+// Moves upwards to the largest radius in full steps
+void moveToFirstCircle()
+{
+    setScale(yMotor, WHOLE_STEPS);  
+    digitalWrite(yMotor[DIR_PIN], UP);  
+    for (int i = 0; i < circleRadius[0]; i++)
```

Comment:
In C, it's more standard to declare loop vars at the top of the scope. Can you declare this before setScale with a newline separating them?